source/common/tracing/common_values.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <string>
+
+#include "common/singleton/const_singleton.h"
+
+namespace Envoy {
+namespace Tracing {
+
+/**
+ * Tracing tag names.
+ */
+class TracingTagValues {
+public:
+  // OpenTracing standard tag names.
+  const std::string Component = "component";
```

Comment:
Not sure if we can make all of these as constexpr. WDYT?

Reply:
These values are separated from http_tracer_impl.h derectly. I prefer to add a TODO, and then complete the const to constexpr migration in a new PR.

Reply:
sgtm